tests/testthat/_snaps/inherits.md

```diff
@@ -0,0 +1,16 @@
+# checks that input is a class
+
+    Code
+      R7_inherits(1:10, "x")
+    Error <simpleError>
+      `class` is not an <R7_class>
+
+# throws informative error
+
+    Code
+      foo1 <- new_class("foo1")
+      foo2 <- new_class("foo2")
+      check_R7_inherits(foo1(), foo2)
+    Error <simpleError>
+      `foo1()` is not an <foo2>
```

Comment:
```suggestion
      `foo1()` is not a <foo2>
```

Should it be `a` rather than `an` in the general error message?

Reply:
Yeah, good catch.

Reply:
Most of the existing uses seem to be followed with "R7" so an is mostly correct.